src/gui/osutils/winutils/WinUtils.cpp

```diff
@@ -79,6 +96,41 @@ bool WinUtils::nativeEventFilter(const QByteArray& eventType, void* message, lon
     case WM_HOTKEY:
         triggerGlobalShortcut(msg->wParam);
         break;
+    case WM_NCCREATE:
+    case WM_ENTERIDLE: // modal dialog box i.e.: save file dialog
+        if (not allowScreenCapture()) {
```

Comment:
We don't use verbs

Reply:
Should I rename to `isScreenCaptureAllowed`?

Reply:
Sorry I meant we don't use `not` we use `!`

Reply:
Oh  ok, I'll refactor the code.

Reply:
Done